cellmlmanip/parser.py

```diff
@@ -208,6 +208,8 @@ def _add_units(self, model):
             # unit is defined in terms of known units - ok to add to model
             if add_now:
                 definition = self._make_pint_unit_definition(unit_name, unit_elements)
+                if self.model.units.is_defined(unit_name):
+                    raise ValueError(f'Duplicate unit definition {unit_name}, unit definitions need to be unique!')
```

Comment:
```suggestion
                    raise ValueError(f'Duplicate unit definition {unit_name}, unit names need to be unique')
```

Reply:
You can have `1 michael = 1 meter / second` and `1 maurice = 1 meter / second`. Don't want people to get confused :D